src/chainstate/coordinator/mod.rs

```diff
@@ -306,7 +306,15 @@ impl<'a, T: BlockEventDispatcher, U: RewardSetProvider> ChainsCoordinator<'a, T,
     ) -> ChainsCoordinator<'a, T, (), U> {
         let burnchain = burnchain.clone();
 
-        let mut boot_data = ChainStateBootData::new(&burnchain, vec![], None);
```

Comment:
Can you add a constructor like `::new_without_vesting` that could be used in all the places where `ChainStateBootData::new(` used to be invoked?

Reply:
Done, restored original usage of the `::new` constructor.